backend/django/core/utils/utils_model.py

```diff
@@ -56,6 +56,9 @@ def cohens_kappa(project):
     labels_seen = set()
     for d in irr_data:
         d_log = IRRLog.objects.filter(data=d, data__project=project)
+        # NOTE: Need to figure out why tests fail when this is not there
+        print(list(d_log), len(d_log))
```

Comment:
Is this something you still plan on looking into? Or need another set of eyes on?  I'd like to have this figured out instead of leaving as a TODO

Reply:
I put the comment there in case you had some ideas. I'd like to fix this before merging, but I'm not sure what could be the cause of it.

Reply:
I've addressed this in bb58620